src/components/Comments/WriteComment.js

```diff
@@ -68,7 +68,7 @@ const WriteComment = ({
     if (isKeyboardOpen) {
       commentInputRef.current.focus();
     }
-  }, [isKeyboardOpen, commentInputRef.current]);
+  }, [isKeyboardOpen, commentInputRef?.current]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining (`?.`) in the dependency array of `useEffect` is a good practice. It prevents potential errors if `commentInputRef.current` is undefined when the effect runs. This change improves the robustness of the code.
```diff
- }, [isKeyboardOpen, commentInputRef.current]);
+ }, [isKeyboardOpen, commentInputRef?.current]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->